### ISSUE TITLE
[BUG] wasm-ld: Error  : --shared-memory is disallowed by libHarfBuzzSharp.hb-common.o because it was not compiled with 'atomics' or 'bulk-memory' features.

### ISSUE THREAD

**Issue by wieslawsoltes** [mono/SkiaSharp#2285]
**Description**
```
wasm-ld: Error  : --shared-memory is disallowed by libHarfBuzzSharp.hb-common.o because it was not compiled with 'atomics' or 'bulk-memory' features.
```

Trying to build .NET 7 app with browser-wasm RID and   <WasmEnableThreads>true</WasmEnableThreads> based on:
https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-7-rc-2/#webassembly-multithreading-experimental

**Code**

**Expected Behavior**

It builds

**Actual Behavior**

Linker error

**Basic Information**

- Version with issue: 2.88.3, 2.8.2.3
- Last known good version:  
- IDE:  
- Platform Target Frameworks: .NET 7 RC2
  - Android:  
  - iOS:  
  - Linux:  
  - macOS:  
  - Tizen:   
  - tvOS:  
  - UWP:   
  - watchOS:  
  - Windows Classic:  
- Target Devices:   
  - 
  - 
  
<details>
  <summary>Detailed IDE/OS information (click to expand)</summary>
  
```

PASTE ANY DETAILED VERSION INFO HERE

```
</details>

**Screenshots**

**Reproduction Link**

**Comment by mattleibow** [comment]
Thanks for reporting! @jeromelaban is a hero on the case!

**Comment by wieslawsoltes** [comment]
> Fixed in #2286

@mattleibow Is the fix included in 2.8.2.4-preview.7 nightly build ?

If it is I still get that error.

**Comment by Numpsy** [comment]
Fwiw, I've managed to get an Avalonia based app compiing with ```WasmEnableThreads``` set to true using the preview versions of SkiaSharp/HarfBuzzSharp, though I had to change the Avalonia.Web targets files to use a different version of the binaries to do it and haven't managed to get something running stabely yet
